deadlines/src/main/java/com/palantir/deadlines/Deadlines.java

```diff
@@ -158,12 +183,12 @@ public static <T> void parseFromRequest(
     }
 
     private static void storeDeadline(long deadline, boolean internal) {
-        checkExpiration(deadline, internal);
-        ProvidedDeadline providedDeadline = new ProvidedDeadline(deadline, getClockNanoTime(), internal);
+        checkExpiration(deadline, internal, false);
+        ProvidedDeadline providedDeadline = new ProvidedDeadline(deadline, getClockNanoTime(), internal, false);
         deadlineState.set(providedDeadline);
     }
 
-    private static void checkExpiration(long deadline, boolean internal) {
+    private static void checkExpiration(long deadline, boolean internal, boolean _disablePropagation) {
```

Comment:
should we encode `disablePropagation` as a separate tag on the meter?

Reply:
I think that would be helpful